lib/registry/src/config.rs

```diff
@@ -36,16 +36,32 @@ pub struct Proxy {
     pub url: Option<String>,
 }
 
-#[derive(Deserialize, Serialize, Debug, PartialEq, Eq, Default)]
+#[derive(Deserialize, Serialize, Debug, PartialEq, Eq)]
 pub struct UpdateNotifications {
     pub enabled: String,
 }
 
-#[derive(Deserialize, Serialize, Debug, PartialEq, Eq, Default)]
+impl Default for UpdateNotifications {
+    fn default() -> Self {
+        UpdateNotifications {
+            enabled: "false".to_string(),
+        }
+    }
+}
+
+#[derive(Deserialize, Serialize, Debug, PartialEq, Eq)]
 pub struct Telemetry {
     pub enabled: String,
```

Comment:
Should just be a bool

Reply:
Switched to a new key so it doesn't break wapm-cli